crates/ra_project_model/src/lib.rs

```diff
@@ -263,7 +264,11 @@ impl ProjectWorkspace {
                             json_project::Edition::Edition2018 => Edition::Edition2018,
                         };
                         let cfg_options = {
-                            let mut opts = CfgOptions::default();
+                            let target = krate.target.as_deref();
```

Comment:
If the crate doesn't have a target specified, should I fallback to the target passed in to this fn? @woody77